example.py

```diff
@@ -0,0 +1,15 @@
+from nameko.rpc import rpc
+from nameko_twilio import Twilio
+
+
+class SmsService:
```

Comment:
On second thoughts, this should probably be called `MessageService`. We don't want to tie a technology name (SMS) to our service name. 
What if tomorrow we wanted to start sending MMS?